charts/nifi-efs/values-dev.yaml

```diff
@@ -33,14 +33,14 @@ ingress:
     nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
     nginx.ingress.kubernetes.io/upstream-vhost: "localhost:8443"
     nginx.ingress.kubernetes.io/proxy-redirect-from: "https://localhost:8443"
-    nginx.ingress.kubernetes.io/proxy-redirect-to: "https://nifi3.ai-cdc-nbs.eqsandbox.com"
+    nginx.ingress.kubernetes.io/proxy-redirect-to: "https://nifi.ai-cdc-nbs.eqsandbox.com"
```

Comment:
for the dev environment the url should be https://nifi.dev-cdc-nbs.eqsandbox.com. Fix needed on lines 36, 39, 41, and 43 (maybe more, please double check).